# def user_defined_exception():
#     number = 18
#
#     try:
#         input_num = int(input("Enter a number: "))
#
#         else:
#             print("Eligible to Vote")
#
#     except:
#         print("Exception occurred: Invalid Age")
#
#
# if __name__ == '__main__':
#     user_defined_exception()
